8/Dockerfile

```diff
@@ -0,0 +1,47 @@
+# This file is generated from ../Dockerfile.template.
+# To update this file run go to repo root and run './template.sh 8'.
+FROM node:8-alpine
+
+# @see http://label-schema.org/rc1/
+LABEL maintainer="Phase2 <outrigger@phase2technology.com>" \
+  # Replacement for the old MAINTAINER directive has fragmented.
+  # "vendor" prevents CentOS from leaking through, the other is for tools integrations.
+  vendor="Phase2 <projectname@phase2technology.com>" \
+  org.label-schema.vendor="Phase2 <projectname@phase2technology.com>" \
+  name="Node v8" \
+  org.label-schema.schema-version="1.0"
+
+# Operational node systems commonly need to make remote requests.
+RUN apk --update --no-cache add bash curl git openssl tini
+
+ENV \
+  OUTRIGGER_QUIET=0 \
```

Comment:
Just invented this to control whether a new pre-start script should output anything.